dual_net.py

```diff
@@ -306,8 +315,10 @@ def validate(working_dir, tf_records, checkpoint_name=None, validate_name=None):
     checkpoint_name = checkpoint_name or estimator.latest_checkpoint()
 
     def input_fn():
-        return preprocessing.get_input_tensors(FLAGS.train_batch_size, tf_records,
-                                               shuffle_buffer_size=20000, filter_amount=0.05)
+        return preprocessing.get_input_tensors(
+            FLAGS.train_batch_size, tf_records, filter_amount=0.05,
```

Comment:
This should probably be updated to filter_amount=1.0 to match train() (see https://github.com/tensorflow/minigo/blob/master/dual_net.py#L292)